mappings/net/minecraft/resource/NamespaceResourceManager.mapping

```diff
@@ -23,10 +23,60 @@ CLASS net/minecraft/class_3294 net/minecraft/resource/NamespaceResourceManager
 		ARG 1 id
 	METHOD method_24233 addPack (Lnet/minecraft/class_3262;)V
 		ARG 1 pack
+	METHOD method_41256 addPack (Lnet/minecraft/class_3262;Ljava/util/function/Predicate;)V
+		ARG 1 pack
+		ARG 2 filter
```

Comment:
Hmm. I named that as "allowedPathPredicate" when mapping bottom-up (from usage) and as "filter" when mapping top-down (from passing), so it's possible there are some inconsistencies. (In this case I think both makes sense, because this is the "pack filter" - but I don't think there is a term for the negated version, thus the generic name of `allowedPathPredicate`.)